script/ModuleRegistry.s.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.21;
+
+import { Script } from "forge-std/Script.sol";
+import { ModuleRegistry } from "../src/ModuleRegistry.sol";
+
+contract ModuleRegistryScript is Script {
```

Comment:
Can you add a script in the `package.json` to call this deployment from a pnpm command?
FYI, we already have a `deploy:SchemaRegistry:goerli` command 😉